### ISSUE TITLE
[Feature Request] Verbose output for daemon

### ISSUE THREAD

**Issue by mgutt** [hoytech/vmtouch#86]
I like to see what the daemon is doing, but this command returns nothing:
`vmtouch -v -d -l -p 0M-60M "/mnt/disk1/Movie/OP"`

I like to see following things:
- what has been crawled
- how many data of each file has been crawled
- when it finished to crawl

**Comment by hoytech** [comment]
Yeah daemon was really only intended to keep the lock active. I think actually adding daemon mode to vmtouch wasn't a good idea, and it's sort of deprecated.
